zlib-flate: add page (#7839)
tldr-pages/tldr: pages/common/zlib-flate.md

# zlib-flate

> Raw zlib compression and decompression program.

> > The zlib-flate program is part of the qpdf package.

Reviewer: We use this format
```suggestion
> Part of `qpdf`.
```

> More information: <https://manned.org/zlib-flate>.

- Compress a file:

> `zlib-flate -compress < {{file}} > {{compressed.zlib}}`

Reviewer 2: ```suggestion
`zlib-flate -compress < {{path/to/input_file}} > {{path/to/compressed.zlib}}`
```

- Uncompress a file:

> `zlib-flate -uncompress < {{compressed.zlib}} > {{file}}`

Reviewer 2: ```suggestion
`zlib-flate -uncompress < {{path/to/compressed.zlib}} > {{path/to/input_file}}`
```

Author: Thanks for the suggestions!
In this case i think `output_file` is more appropriate. Already updated.

> - Compress a file with a specified compression level [0-9]:

Reviewer: Based on `gzip` page
```suggestion
- Compress a file with a specified compression level. 1=Fastest (Worst), 9=Slowest (Best):
```

Author: I did some tests to check about the level 0 (which apparently means "no compression at all") and turns out using the levels (besides 0) didn't change the size of the resulting file... I don't have any VMs right now, but would be nice if someone could test this on another machine to verify if the same thing occurs

Reviewer: Yeah, I checked `0` is just save.
```suggestion
- Compress a file with a specified compression level. 0=Fastest (Worst), 9=Slowest (Best):
```

> `zlib-flate -compress={{compression_level}} < {{file}} > {{compressed.zlib}}`

Reviewer 2: ```suggestion
`zlib-flate -compress={{compression_level}} < {{path/to/input_file}} > {{path/to/compressed.zlib}}`
```
